python_ta/debug/accumulation_table.py

```diff
@@ -90,24 +90,30 @@ def __init__(
 
         Args:
             accumulation_names: a list of the loop accumulator variable names to display.
-
+            output: optional filepath where the table will be written. If None, prints to stdout.
+            format: output format, either "table" (formatted table) or "csv" (CSV format).
         """
-        self.loop_accumulators = {accumulator: [] for accumulator in accumulation_names}
-        self.loop_variables = {}
-        self._loop_lineno = 0
+        self._accumulator_names = accumulation_names
```

Comment:
The behaviour of `accumulation_names` should probably be broadened as well.

The current behaviour (a list of strings that's applied to _all_ loops) is good.

A second option that I'd like you to add is that when `accumulation_names` is a `list[list[str]]`, then each inner list should correspond to the accumulation names for the corresponding loop.